python/ruff-ecosystem/ruff_ecosystem/check.py

```diff
@@ -153,6 +153,9 @@ def markdown_check_result(result: Result) -> str:
                     # skip display. This shouldn't really happen and indicates a problem in the
                     # calculation of these values. Instead of skipping entirely when `total_changes`
                     # is zero, we'll attempt to report the results to help diagnose the problem.
+                    #
```

Comment:
This comment is kind of hard to discover (when reading `max_display_per_rule`). I suggest to move the comment to line 168 and refer to the explanation in this comment.